server/server.js

```diff
@@ -12,6 +13,15 @@ const PORT = 3000;
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+/**
+ * session middleware
+ */
+app.use(session({
+  secret: 'mysuperduperubersecretsecret',
```

Comment:
mayyyybeeeee we can put this as an `env` variable?

Reply:
fixed. will be included in next commit.